guide/14-deep-learning/point_cloud_segmentation_using_pointcnn.ipynb

```diff
@@ -18,7 +18,7 @@
    "cell_type": "markdown",
```

Comment:
<ol><li>Left <s>Image</s> <strong>image </strong>shows a raw XYZ lidar points whereas <strong>the</strong> right image shows the same data labelled&nbsp;</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/713/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
''corrected accordingly'', Thanks.